content/blog/getting-started-with-the-hpe-data-services-cloud-console-public-rest-api.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Getting Started with the HPE Data Services Cloud Console Public REST API
+date: 2021-12-15T16:01:03.611Z
+priority: 2
+author: Ron Dharma
+authorimage: https://gravatar.com/avatar/8102f2adeef249065ccb9c43b8866d17?s=192
+thumbnailimage: /img/dscc-icon.png
+tags:
+  - data-services-cloud-console
+---
+![Unified DataOps ](/img/dscc-idp-core-architect.png "Unified DataOps")
+
+Customers across industries are struggling with the complexity of managing data and infrastructure, because it creates a roadblock to innovation and agility. Today, every organization is required to unleash the power of data to drive digital transformation, but fragmented data management tools, manual processes, and infrastructure silos - spanning edge to cloud - are getting in the way. This complexity also amplifies business risk, and it's only getting harder as data continues to grow, apps evolve, and infrastructure continues its spread from edge to cloud. 
+
+Data Services Cloud Console public REST API provides a resource for customers who are looking to enhance their infrastructure management and data-ops using the programmatic extensions from Data Services Cloud Console (DSCC).
+
+### A Public REST API which is based on the OpenAPI 3.X Specification
+
+![API diagram](/img/universal-public-api.png "API ")
+
+Hewlett Packard Enterprise (HPE) offers the DSCC public REST API to provide the agility previously mentioned. It is specified based on the OpenAPI format version 3 (OpenAPI 3.0 information). The specification defines a standard, language-agnostic interface to the RESTful API allowing clients (both human and computer) to consume capabilities of DSCC services efficiently. The API definition is available for download in either OpenAPI 3 YAML or JSON format at the link mentioned in at the next chapter.
```

Comment:
Remove the word "at" just prior to "the next chapter" so it reads "... at the link mentioned in the next chapter."